app/flags.go

```diff
@@ -55,6 +56,12 @@ var (
 		EnvVars:  []string{"TEMPORAL_CLOUD_NAMESPACE"},
 		Required: true,
 	}
+	RegionFlag = &cli.StringFlag{
+		Name:     RegionFlagName,
+		Usage:    "The region to use for the request, if not set the server will use the namespace's region",
+		Aliases:  []string{"re"},
+		Required: false,
+	}
```

Comment:
Should this flag be defined in the other file closer to the request instead? I wasn't sure whether the convention was to define it here or in the other file

Reply:
Can we add this flag as sinkRegion specifically? Then it could be added to app/namespace.go